include/okapi.h

```diff
@@ -74,3 +74,7 @@ int32_t oberon_verify_proof(struct ByteBuffer request,
 void didcomm_byte_buffer_free(struct ByteBuffer v);
 
 void didcomm_string_free(char *s);
+
+void okapi_bytebuffer_free(struct ByteBuffer v);
+
+void okapi_string_free(char *s);
```

Comment:
rename, or additional function headers?

Reply:
Rename, when it makes sense. Just deprecating them now, since they have the old library naming. Won't remove them soon.

Reply:
https://github.com/trinsic-id/okapi/issues/262